src/titiler/core/titiler/core/templates/map.html

```diff
@@ -86,20 +86,33 @@
 
     let bounds = [...data.bounds]
 
+    var left = bounds[0],
+      bottom = bounds[1],
+      right = bounds[2],
+      top = bounds[3];
+
+    if (left < right) {
+      left = Math.max(left, {{ tms.bbox.left }})
+      right = Math.min(right, {{ tms.bbox.right }})
+    }
+    bottom = Math.max(bottom, {{ tms.bbox.bottom }})
+    top = Math.min(top, {{ tms.bbox.top }})
```

Comment:
I needed to do this because some NetCDF data have bounds that exceed the TMS bbox